torch/_logging/_registrations.py

```diff
@@ -135,7 +135,7 @@
 )
 register_artifact(
     "not_implemented",
-    "Prints log messages whenever we return NotImplemented in a multi-dispatch, letting you trace through each object we attempted to dispatch to",
+    "Prints log messages whenever we return NotImplemented in a multi-dispatch, letting you trace through each object we attempted to dispatch to"
```

Comment:
Is this suggested by the internal linter? This is contradictory to Github CI.

Reply:
it is auto-generated by agent.